PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -1276,6 +1293,19 @@ def PrepJMECustomNanoAOD(process,runOnMC):
     recoJetInfo = recoJA.addRecoJetCollection(process, **cfg)
     AddNewPatJets(process, recoJetInfo, runOnMC)
 
+  # Create list of discriminatorSources for Run 3 excluding btagCSVV2
+  discriminatorSourcesPUPPI_run3 = copy.copy(process.updatedPatJetsTransientCorrectedAK4PFPUPPIFinal.discriminatorSources)
+  discriminatorSourcesPUPPI_run3.pop(0)
+  discriminatorSourcesCHS_run3 = copy.copy(process.updatedPatJetsTransientCorrectedAK4PFCHSFinal.discriminatorSources)
+  discriminatorSourcesCHS_run3.pop(0)
+  run3_common.toModify(
+    process.updatedPatJetsTransientCorrectedAK4PFPUPPIFinal,
+    discriminatorSources = discriminatorSourcesPUPPI_run3
+  ).toModify(
+    process.updatedPatJetsTransientCorrectedAK4PFCHSFinal,
+    discriminatorSources = discriminatorSourcesCHS_run3
+  )
```

Comment:
Imho, the change in #40531 is cleaner wrt this one because it applies the modifier only once, changing the _inputs_ of the customisation, as opposed to its output products. The advantage is that, unlike here,
  1. there is no dependence on the the internal naming of these output collections (e.g. `TransientCorrected*Final`);
  2. unnecessary objects (CSVv2 inputs) are not added to Tasks/Sequences (in this PR, I don't think the Tasks/Sequences would be fully cleaned up for era>=Run3);
  3. in #40531 the modifier is applied only once, not once per customisation call (i.e. twice in this case).

It's true that #40531 is only a minimal fix, and this PR does more (modulo my comment above). I think this PR should go 'on top' of #40531, rather than being alternative.

What is clear is that there is a fix for `11634.15` for the next IB, which is all I was aiming for. The rest is up to reviewers.

Reply:
I agree #40531 is cleaner, could it be included here instead of the above?

Reply:
addessed in [cc2047c](https://github.com/cms-sw/cmssw/pull/40539/commits/cc2047c89771ad2cfb6fdb3709ead4bae4601c0c)